bin/ci

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
 cd "$(dirname "$(dirname $0)")"
-sbt ci
+command="${1:-all}"
```

Comment:
I have no idea what this does, but if the branch passes I won't argue with it.

Reply:
https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts